tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -611,7 +611,7 @@ generic_config_updater/test_ecn_config_update.py::test_ecn_config_updates:
       - "topo_type in ['m0', 'mx']"
       - "release in ['202211']"
 
-generic_config_updater/test_eth_interface.py::test_eth_interface_admin_change:
```

Comment:
Do you still need to skip old testcase based on old issue?

Reply:
@qiluo-msft yes, the issue is same, actually, it's caused by test_replace_fec not test_eth_interface_admin_change. I verified on 7260 t0 testbed.